app/controllers/api/v1/schools_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::SchoolsController < ApplicationController
+
+  # GET /schools
+  def index
+    @schools = School.all
```

Comment:
How many are here?  Do we need pagination?  Will callers want to query for schools?

Reply:
added pagination - but would like a review of if the implementation seems appropriate (using `will_paginate` gem)